scripts/grcov_cxx_qt.sh

```diff
@@ -0,0 +1,5 @@
+export RUSTFLAGS="-Cinstrument-coverage"
```

Comment:
can you add the bash shebang and license (SPDX info copy from other files and use your name) to the top

```bash
#!/usr/bin/env bash

# SPDX-FileCopyrightText: ...
# SPDX-FileContributor: ...
#
# SPDX-License-Identifier: MIT OR Apache-2.0
```